index.html

```diff
@@ -1,70 +1,113 @@
-      </li>
-      <li>
-        <a href="challenge/mail-icon.png"><img src="challenge/mail-icon.png"></a>
-      </li>
-    </ul>
-  </nav>
-</header>
-  <section>
-    <h1>Chocolate Pizza</h1>
-    <h5>Posted on 15 Dec 2013 / Desserts</h5>
-    <img src="challenge/print-icon.png"/>
-  </section>
-  <section>
-    <a href="choco-pizza.png"><img src="challenge/choco-pizza.png"></a>
-  </section>
-  <section>
-    <article>
-      <p> For the fig-swirl: Melt butter over medium hear in a saucepan. Add brown sugar and stir to dissolve. Halve all the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
-      <p>Ice cream: In a small pot over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
-      <p>Using an ice cream machine, pour liquids into the frozen basen and process according to manufacturer instructions, i.e., let spin and thinken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
-    </article>
-  </section>
-  <section>
-    <a href="challenge/list-bg.png"><img src="challenge/list-bg.png"></a>
-  </section>
-  <section>
-    <aside><a href="van-pic"></a></aside>
-    <p>Vanessa Stevenson</p>
-    <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
-    <aside>share recipe</aside>
-  </section>
-<footer>
-  <a href="challenge/small-logo.png"><img src="challenge/small-logo.png">fork and knife logo</a>
-  <ul>
-    <li>Delicious &copy 2014 - All Rights Reserved</li>
-    <li>Proudly published with Ghost.</li>
-  </ul>
-</footer>
-</body>
-</html>
\ No newline at end of file
+<html class="no-js">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>
+      Chocolate Pizza - Delicious: THE BEST FOOD BLOG ON THE WEB.
+    </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='http://fonts.googleapis.com/css?family=Abel|Patrick+Hand|Yellowtail|Lobster|Oswald:400,700' rel='stylesheet' type='text/css'>
+    <link href="base.css" type="text/css" rel="stylesheet"/>
+    <link href="layout.css" type="text/css" rel="stylesheet"/>
+    <link href="module.css" type="text/css" rel="stylesheet"/>
+  </head>
+  <body>
+    <div class="pageContainer">
+      <main>
+        <header class="section group">
+          <section class="logo col span_6_of_12 ">
+            <img src="challenge/logo.png"></a>
+            <h1>Delicious</h1>
+            <h2>THE BEST FOOD BLOG ON THE WEB</h2>
```

Comment:
Think about changing this to standard casing. You can use css with `text-transform: uppercase` to get this effect.